apps/cowswap-frontend/src/common/updaters/orders/ExpiredOrdersUpdater.ts

```diff
@@ -33,26 +32,24 @@ export function ExpiredOrdersUpdater(): null {
       try {
         isUpdating.current = true
 
-        // Filter orders:
+        // Filter expired orders:
+        // - Only eth-flow orders
         // - Owned by the current connected account
-        // - Created in the last 5 min, no further
         // - Not yet refunded
-        const pending = expiredRef.current.filter(({ owner, creationTime: creationTimeString, refundHash }) => {
-          const creationTime = new Date(creationTimeString).getTime()
+        const orderWithoutRefund = expiredRef.current.filter(({ owner, refundHash, sellToken }) => {
+          const isEthFlowOrder = sellToken === NATIVE_CURRENCY_BUY_ADDRESS
 
-          return (
-            owner.toLowerCase() === lowerCaseAccount && now - creationTime < EXPIRED_ORDERS_PENDING_TIME && !refundHash
-          )
+          return isEthFlowOrder && owner.toLowerCase() === lowerCaseAccount && !refundHash
```

Comment:
What about regular orders?
Now they are not checked at all.

I meant to keep the former behaviour for regular orders, and check ethflow orders without a refund hash indefinitely.

Reply:
But only eth-flow order might have refundHash, isn't it?

Reply:
Yes, you are 100% correct.
I was confusing this updated with the `CancelledOrdersUpdater`